src/specs/uniswap/readme.md

```diff
@@ -0,0 +1,44 @@
+# Spec for Lido Bridge
+
+## What does the bridge do? Why build it?
+
+The bridge swaps `_inputAssetA` for `_outputAssetA` along the path specified in `_auxData`.
+Encoding of a path allows for up to 2 split paths (see documentation in UniswapBridge.sol for details) and up to 3 pools (2 middle tokens) in each split path.
+
+## What protocol(s) does the bridge interact with ?
+
+The bridge interacts with [Uniswap v3](https://docs.uniswap.org/protocol/introduction).
+
+## What is the flow of the bridge?
+
+There is only 1 flow in the bridge.
+In this flow `inputAssetA` is swapped for `outputAssetA` along the path specified in `_auxData`
+
+**Edge cases**:
+
+- Interaction might revert in case the minimum acceptable price is set too high or the price of the output asset moves up before the interaction settles.
+- Users might get sandwiched in case the minimum acceptable price is set too low or the price of the output token goes down before the interaction settles making the users lose positive slippage.
+
+### General Properties for both deposit and withdrawal
```

Comment:
Not sure you would call it deposit and withdrawal when there is no distinction here. Might just call it convert or swap?